src/workflows/bidsFFX.m

```diff
@@ -73,14 +73,16 @@ function bidsFFX(action, opt, funcFWHM)
 
           saveAndRunWorkflow(matlabbatch, batchName, opt, subID);
 
-          plot_power_spectra_of_GLM_residuals( ...
-                                              getFFXdir(subID, funcFWHM, opt), ...
-                                              opt.metadata.RepetitionTime);
+          if opt.glmQA.do
+            plot_power_spectra_of_GLM_residuals( ...
+                                                getFFXdir(subID, funcFWHM, opt), ...
+                                                opt.metadata.RepetitionTime);
 
-          deleteResidualImages(getFFXdir(subID, funcFWHM, opt));
+            deleteResidualImages(getFFXdir(subID, funcFWHM, opt));
 
-          movefile(['sub-', subID, '_task-', opt.taskName, '_design_*'], ...
-                   getFFXdir(subID, funcFWHM, opt));
+            movefile(['sub-', subID, '_task-', opt.taskName, '_design_*'], ...
```

Comment:
this line will move the `*design*` images in the stats folder, so those images are created in `plot_power_spec...`?

Reply:
correct